### DISCUSSION TITLE
Hypocritical Policy

### DISCUSSION THREAD

**Discussion by Nazrac** [sn4k3/UVtools#442] [General]
You say you are against closed source privatized systems because it hurts the end users. You are a bunch of hypocrites for doing this. The only people you are causing annoyance to is the user who wished to modify their files. Do you think that doing this is going to stop companies from using CTB boards? Do you think that Chitu will go back open source. Its there product they have every right to do this. I bet half of you are looking at this on apple products and talk about how great they are, guess what, closed source. You say its closed source yet you have the SDK so what are you really mad about? Lychee supports it, CTB just wants royalties for the PRO sales, what are you complaining about? Nobody complains about paid software from them yet you complain about Chitu Pro, Get off your high horse and fine put the message about how much you hate chitubox but quit making my life difficult based on your opinion.

**Comment by sn4k3** [comment]
**No** is the anwser to all your questions but clearly you don't know what you talking about nor in-depth of the actual problem.

> The only people you are causing annoyance to is the user who wished to modify their files.

Sorry, you have to explain this better, sometimes I really can't follow enlightenment people... If CTB is owned by Chitu, why a small hobby third-party project or me have such obligation/power? Isn't that a flame to put on thier side? I mean, they made the car, didn't gave you the key, what do i have anything with that? Is not thier responsability to give you all the tools to support thier inventions in the first place? How do i annoy users by not support something that isn't mine nor responsible by, interesting 🤔

> It's there product they have every right to do this.

Your countersense kills me 🤣 
Why UVtools don't have same rights? 😭 

> but quit making my life difficult based on your opinion.

You speak like i had any obligations here or own something to anyone, bad news: I don't and it fells good. I can do whatever i want with my own project, no? If it cause such hard time why use it? Oh even better, you can modify it to be like you, so why not modify it to your needs? Ah, wait i know this one... because Chitu don't and never will allow such bad pratice, and they are the good guys with all the rights here. I get it now.
Also don't forget, next time your resin ends, you can put it on me, i will apology for the inconvenience i caused, i mean, must be my incovenience right?

**Although, while i was replying i was wondering:**

1. isn't UVtools compatible with latest CTB and all thier stuff and even more? (Oh look it can edit stuff which even Chitubox Pro can't, i'm really an reputable bad ass villan)
2. isn't UVtools free?
3. Yes and yes, so what about is this guy complaining?

Really, next time you dislike something either do yourself better alternative, or raise proper arguments - `my life difficult` is just lame and selfish.

BTW, if you have the time, make sure to read this: https://github.com/sn4k3/UVtools/discussions/440 we need funding! Help our Apple fellows and half of our community.

**Comment by wjsteele** [comment]
> "You say you are against closed source privatized systems because it hurts the end users. You are a bunch of hypocrites for doing this. The only people you are causing annoyance to is the user who wished to modify their files. Do you think that doing this is going to stop companies from using CTB boards? Do you think that Chitu will go back open source. Its there product they have every right to do this. I bet half of you are looking at this on apple products and talk about how great they are, guess what, closed source. You say its closed source yet you have the SDK so what are you really mad about? Lychee supports it, CTB just wants royalties for the PRO sales, what are you complaining about? Nobody complains about paid software from them yet you complain about Chitu Pro, Get off your high horse and fine put the message about how much you hate chitubox but quit making my life difficult based on your opinion."

## Let's break down your own comments.

**"You say you are against closed source privatized systems because it hurts the end users."**  Where did they say that?

**"You are a bunch of hypocrites for doing this."**  Hypocrite: someone who claims to have moral standards or beliefs to which one's own behavior does not conform.  How does that apply here?  There is nothing in their statements which I can find that indicates this.

**"The only people you are causing annoyance to is the user who wished to modify their files."**  How is giving users a tool to edit their own files an annoyance?  Please explain.

**"Do you think that doing this is going to stop companies from using CTB boards? Do you think that Chitu will go back open source. Its there product they have every right to do this."**  That's not the intent of the project, and two, they never were open source.  The intent of this project is to give users a tool to allow conversion and editing of both open and closed source packages output files, beyond the capabilities of the original software used to create them.

**"I bet half of you are looking at this on apple products and talk about how great they are, guess what, closed source."**  So... not sure what that has to do with anything.  The intent of this project is as stated above.

**"You say its closed source yet you have the SDK so what are you really mad about?"**  The SDK requires a license and does not give full access to all the available features needed.  Also, Chitu isn't the only slicer or electronics supported by this tool.  Why do you think it's only about them?

**"Lychee supports it, CTB just wants royalties for the PRO sales, what are you complaining about?"**   A) Good for Lychee... B) So, that's their right, C) Who's complaining?  This tool goes beyond a single companies products, and allows for features that no single slicer has.

**"Nobody complains about paid software from them yet you complain about Chitu Pro, Get off your high horse and fine put the message about how much you hate chitubox but quit making my life difficult based on your opinion."**  How does any of this make your life difficult?    Just go away and don't worry about it... problem solved.  Honestly, I don't care one red cent about Chito Pro or their business model... I use this tool to allow for features and editing that I can't get anywhere else.  My preferred slicer is PrusaSlicer, and I'd like to print on all my printers... this tool allows me to do that very easily regardless of their platform.

**Comment by Nazrac** [comment]
I am glad you decided to allow people to modify the evil encrypted format. No I am not a troll from China, I am an American served in the Navy and I support companies doing whatever they want. I will I am glad you probably realized that it really want hurting anyone but the end user and lets not get confused they did openly say they would give out the SDK for non-profits for free. I know that this community is a bunch of self righteous pompous millennials but you know you did the right thing here. You give your opinion every time I open a file but that's fine. Have a nice day.
